### ISSUE TITLE
ComposeModifierMissing shouldn't error on preview functions

### ISSUE THREAD

**Issue by ZacSweers** [slackhq/compose-lints#260]
Currently it does

**Comment by ZacSweers** [comment]
Nvm can't repro now

**Comment by KatieBarnett** [comment]
@ZacSweers I can reproduce this when I have a Preview annotation that has been made up of other preview configurations similar to:

```
@Preview(group = GROUP_NORMAL, device = DEVICE_STANDARD_PHONE)
@Preview(group = GROUP_DEVICE_SMALL_PHONE, device = DEVICE_SMALL_PHONE)
annotation class MyScreenPreviews
```

Interestingly this doesn't happen to all previews, I am yet to find the exact reason, when I replace this custom annotation with `@Preview` the error goes away. It does not seem to matter if the previews have a preview parameter or not.

**Comment by ansman** [comment]
Fixed in #494
